toolkit/tools/imagecustomizerapi/systemconfig.go

```diff
@@ -56,5 +57,13 @@ func (s *SystemConfig) IsValid() error {
 		}
 	}
 
+	allServices := append(s.Services.Enable, s.Services.Disable...)
```

Comment:
Considering that you have a separate `Services` struct, you could move this code into a new `IsValid()` function for the `Services` struct.